src/OpenDiffix.Core/Executor.fs

```diff
@@ -96,7 +107,19 @@ let private executeAggregate executionContext (childPlan, groupingLabels, aggreg
     |> Array.iteri (fun i aggregator -> aggArgs.[i] |> List.map argEvaluator |> aggregator.Transition)
 
   state
-  |> Seq.map (fun pair -> pair.Value)
+  |> Seq.map (fun pair ->
+    let bucket = pair.Value
+
+    { bucket with
+        // A bucket is low count if any low count aggregator in scope returns true.
+        LowCount =
+          lowCountAggs
+          |> Array.exists (fun aggIndex ->
+            bucket.Aggregators.[aggIndex].Final bucket.ExecutionContext
+            |> Value.unwrapBoolean
+          )
+    }
+  )
```

Comment:
I don't think this approach is optimal.
I would prefer if we uniquely set the low-count aggregator index during query analysis. Or if we implicitly make it to always be the first in the list. Then we can drop it from our custom queries in the desktop client.

Reply:
> Or if we implicitly make it to always be the first in the list.

How do we handle standard queries if we do this?

> Then we can drop it from our custom queries in the desktop client.

How do we pass AID information? If through table config, then queries will be rewritten which we don't want.

Reply:
> How do we handle standard queries if we do this?

Either we always compute it and then ignore it, or this only applies to anonymized queries. Whatever generates less special cases.

> How do we pass AID information?

You're right, if we mark columns as AIDs then we need to add a toggle for query rewriting. Not ideal.
Alternatively, we can run the query twice, once as standard and once as anonymized, then merge the results. Also not ideal.
This requires more thought... For now, I think it is still worth it to put the index in the execution context and set it during the analysis stage.

Reply:
Passing an `int option` for now. We can skip the hooks if there is no LCF - meaning the query is likely standard, or this in-between query from desktop which is technically anonymizing but written manually...